canonical/ubuntu.com
# Update ubuntu lts security support graph with new design

## Done

- Creates a new function `createSupportChart` that creates a graph based on the design [found in this figma file](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?type=design&node-id=871-2991&mode=design&t=6JDKoP4OfQHU60Eo-0). A screenshot comparison can be found below.
- Updates where this graph was used previously to work with the new graph (this means adding a place for the key to attach)

## QA

- Go to /desktop/developers and see that the graph matches the design
- Check that it works on medium and large screens and becomes a table on small screens
- Check the other pages where it is used and see that it doesn't break or anything (ie. /about/releases)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8270

## Screenshots

Desgin:
![image](https://github.com/canonical/ubuntu.com/assets/58276363/e2fe392d-65ed-4e96-a698-dc7216829223)

D3 reproduction:
![image](https://github.com/canonical/ubuntu.com/assets/58276363/29eb352d-0472-4b37-a84c-868225e6f536)
